Find-and-Replace/Replace-text-with-content-control/.NET-Standard/Replace-text-with-content-control/Program.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Syncfusion.DocIO;
+using Syncfusion.DocIO.DLS;
+using System.IO;
+using System.Text.RegularExpressions;
+
+namespace Replace_text_with_content_control
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            using (FileStream fileStreamPath = new FileStream(Path.GetFullPath(@"../../../Data/Input.docx"), FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
```

Comment:
@sharonaumao - Please use common variable name for input file stream in every KB and UG topics. Please refer your previous samples.

Reply:
@Ramaraj-Marimuthu -It is done